src/core/issues-notes.js

```diff
@@ -146,6 +156,40 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
+//derives the text-color-class based on the illumination score of the labelColor
+function deriveTextColorClass(labelColor) {
+  const color = labelColor.substring(1);
+  const rgb = parseInt(color, 16);
+  if (isNaN(rgb)){
+    return
+  }
+  const red = (rgb >> 16) & 0xff;
+  const green = (rgb >> 8) & 0xff;
+  const blue = (rgb >> 0) & 0xff;
+  const illumination = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+  return illumination > 140 ? "light" : "dark";
+}
+
+function createLabels(label) {
+  const textColorClass = deriveTextColorClass(label.color);
+  var labelElement = document.createElement("a");
+  labelElement.classList.add("respec-gh-label");
+  labelElement.style.backgroundColor = `#${label.color}`;
+  labelElement.style.textTransform = "none";
+  labelElement.href = label.url;
+  labelElement.innerText = label.name;
+  labelElement.classList.add(`respec-label-${textColorClass}`);
```

Comment:
The CSS of this class has no role to play as it get's overwritten. How do I fix this? The UI looks very unpleasing without the CSS of this class.